server/shared/storage/interface.ts

```diff
@@ -0,0 +1,21 @@
+export type ContentResponse<ContentType> = { content: ContentType, raw: Buffer };
+
+export type FileListResponse = { path: string };
+
+export type DeleteResponse = { isDeleted: boolean | null };
+
+interface IStorage {
+  getFile(key: string): Promise<ContentResponse<string>>;
+  createReadStream(key: string): NodeJS.ReadableStream;
+  createWriteStream(key: string): Promise<NodeJS.WritableStream> | NodeJS.WritableStream;
```

Comment:
Can we ensure this method has consistent return - `Promise<NodeJS.WritableStream>`?

Reply:
👍